COCOAPODS.md

```diff
@@ -0,0 +1,150 @@
+ along with this project. As a result, importing to Xcode is simplified since there is no need to
+ write corresponding Gradle tasks and Xcode build steps manually.
+ 
+ - When building from Xcode, it is possible to use CocoaPods libraries without manual writing
+ .def files and setting cinterop tool parameters. In this case all required parameters can be
+ obtained from the Xcode project configured by CocoaPods.
+
+For an example of using this feature, refer to the
+[`cocoapods`](https://github.com/JetBrains/kotlin-native/tree/master/samples/cocoapods) sample.
+
+### CocoaPods Gradle plugin
+
+The CocoaPods support is implemented in a separate Gradle plugin: `org.jetbrains.kotlin.native.cocoapods`.
+
+> __Note:__ The plugin is based on the multiplatform project model and requires applying the
+`org.jetbrains.kotlin.multiplatform` plugin. See details about the multiplatform plugin at
+the [corresponding page](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html).
+
+When applied the CocoaPods plugin performs the following actions:
+
+1. Add both debug and release frameworks as output binaries for all iOS and macOS targets.
+2. Create a `podspec` task which generates a [podspec](https://guides.cocoapods.org/syntax/podspec.html)
+file for the given project.
+
+The podspec generated includes a path to an output framework and script phases which automate building
+this framework during a build process of an Xcode project. Some fields of the podspec file can be
+configured using the `kotlin.cocoapods { ... }` code block.
+
+<div class="sample" markdown="1" theme="idea" mode="kotlin">
+
+```kotlin
+// Apply plugins.
+plugins {
+    id("org.jetbrains.kotlin.multiplatform") version "1.3.30"
+    id("org.jetbrains.kotlin.native.cocoapods") version "1.3.30"
+}
+
+// CocoaPods requires the podspec to have a version.
+version = "1.0"
+
+kotlin {
+    cocoapods {
+        // Configure fields required by CocoaPods.
+        summary = "Some description for a Kotlin/Native module"
+        homepage = "Link to a Kotlin/Native module homepage"
+    }
+}
+```
+
+</div>
+
+> __Note:__ the following podspec fields are required by CocoaPods:
+> 
+>  - `version`
+>  - `summary`
+>  - `homepage`
+>
+> A version of the Gradle project is used as a value for the `version` field.
+Fields`summary` and `homepage` can be configured using the `cocoapods` code block.
+
+This podspec file can be referenced from a [Podfile](https://guides.cocoapods.org/using/the-podfile.html)
+of an Xcode project. After that the framework built from the Kotlin/Native module can be used from
+this Xcode project. If necessary, this framework is automatically rebuilt during Xcode build process.
+
+### Workflow
+
+The following steps allows one to import a Kotlin/Native module in an existing Xcode project.
+
+0. [Install](https://guides.cocoapods.org/using/getting-started.html#installation) CocoaPods.
```

Comment:
Make sure that you have CocoaPods [installed](https://guides.cocoapods.org/using/getting-started.html#installation).